src/index.js

```diff
@@ -0,0 +1 @@
+require('./src/server').start()
```

Comment:
`server.js` exporteert geen `start()` functie. Volgens mij kan deze hele file weg, `server.js` als entrypoint is prima en staat al zo geconfigureerd in je `package.json`.